docs/IE_DG/ONNX_Supported_Ops.md

```diff
@@ -1,3 +1,19 @@
+# ONNX format support in the OpenVINO™ {#openvino_docs_IE_DG_ONNX_Supported_Ops}
+
+Beginning from 2020.4 release OpenVINO™ supports to read native ONNX models to nGraph representation.
+`Core::ReadNetwork()` method provides a uniform way to read models from IR or ONNX format, it is a recommended approach to reading models.
+Example:
+
+```cpp
+InferenceEngine::Core core;
+auto network = core.ReadNetwork("model.onnx");
+```
+
+This document describes the list of supported ONNX operations and known limitations of this functionality.
+
+OpenVINO™ doesn't provide a mechanism to specify pre-processing (like mean values subtraction, reverse input channels) for ONNX format.
+If ONNX model contains dynamic shapes for input, please use `CNNNetwork::reshape` method for shape specialization.
```

Comment:
If the specific model is meant, please add the definite article before "ONNX model", i.e. "If the ONNX model contains...", if a general ONNX model is meant, add an indefinite article - "If an ONNX model contains..."

Reply:
Fixed